### PR TITLE
Updates the torsion.in file to properly be configured to handle tscript extension

### PR DESCRIPTION
**Pull request by Areloch**

### CODE REVIEW

**Review by JeffProgrammer — APPROVED** [review]

**Comment by JeffProgrammer** [comment]
@Areloch Does the CMake build system have a variable that sets the TS script extension? Since this is a cmake.in file, we should use that variable instead of hardcoding it if it does. If it doesn't, then this is good to merge.

**Comment by lukaspj** [comment]
Ah yeah that makes sense. It does have such a variable `TORQUE_SCRIPT_EXTENSION`

File: Tools/CMake/template.torsion.in [hunk 1]
```diff
@@ -1,18 +1,14 @@
 <TorsionProject>
 <Name>@TORQUE_APP_NAME@</Name>
 <WorkingDir/>
-<EntryScript>main.cs</EntryScript>
+<EntryScript>main.tscript</EntryScript>
```
Comment (Azaezel) [inline Tools/CMake/template.torsion.in]: `<EntryScript>main.@TORQUE_SCRIPT_EXTENSION@</EntryScript>`

File: Tools/CMake/template.torsion.in [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
 <Folder>core</Folder>
-<Folder>scripts</Folder>
-<Folder>art</Folder>
-<Folder>levels</Folder>
-<Folder>shaders</Folder>
 <Folder>data</Folder>
 <Folder>tools</Folder>
 </Mods>
-<ScannerExts>cs; gui; taml; module;</ScannerExts>
+<ScannerExts>cs; gui; taml; module; tscript;</ScannerExts>
```
Comment (Azaezel) [inline Tools/CMake/template.torsion.in]: `<ScannerExts>@TORQUE_SCRIPT_EXTENSION@; gui; taml; module;</ScannerExts>`

**Pull request merged**